bumble/host.py

```diff
@@ -38,13 +38,97 @@
     HCI_LE_READ_LOCAL_SUPPORTED_FEATURES_COMMAND,
     HCI_LE_READ_SUGGESTED_DEFAULT_DATA_LENGTH_COMMAND,
     HCI_LE_WRITE_SUGGESTED_DEFAULT_DATA_LENGTH_COMMAND,
+    HCI_LE_READ_NUMBER_OF_SUPPORTED_ADVERTISING_SETS_COMMAND,
+    HCI_LE_READ_MAXIMUM_ADVERTISING_DATA_LENGTH_COMMAND,
     HCI_READ_BUFFER_SIZE_COMMAND,
     HCI_READ_LOCAL_VERSION_INFORMATION_COMMAND,
     HCI_RESET_COMMAND,
     HCI_SUCCESS,
     HCI_SUPPORTED_COMMANDS_FLAGS,
     HCI_SYNCHRONOUS_DATA_PACKET,
     HCI_VERSION_BLUETOOTH_CORE_4_0,
+    HCI_INQUIRY_COMPLETE_EVENT,
+    HCI_INQUIRY_RESULT_EVENT,
```

Comment:
You're right, the list is becoming too long. I'll make the change here to just import `hci` and use `hci.XXX` in this file. Eventually, I'll try to push a PR later on to do the same change in other files that also import too many symbols directly. It will make the lines a bit longer, but I think it's a cleaner model.